crates/starknet-os/src/hints/tests.rs

```diff
@@ -158,7 +158,9 @@ pub mod tests {
         vm.add_memory_segment();
         vm.add_memory_segment();
 
-        let ids_data = ids_data![vars::ids::DEPRECATED_TX_INFO];
+        let mut ids_data = HashMap::new();
+        ids_data.insert(vars::ids::DEPRECATED_TX_INFO.to_owned(), HintReference::new(0, 0, false, false));
+
```

Comment:
@whichqua why this change is needed?

Reply:
Because the vm treats `pointers` and `relocalables` differently. This is how to define pointers.